apigen/apigen.gradle.kts

```diff
@@ -23,17 +24,7 @@ tasks.jar.configure {
     }
 }
 
-tasks.withType<JavaCompile>().configureEach {
-    options.encoding = "UTF-8"
-    options.compilerArgs = listOf("-Xlint:all", "-Xlint:-path", "-Xlint:-options", "-Werror")
-}
-
 spotless {
-    java {
-        licenseHeaderFile("$rootDir/gradle/spotless/license.java")
-        googleJavaFormat().aosp()
-    }
-
```

Comment:
I don't think this is wrong but how do these bits happen now?

Reply:
The common plugin does that.